tests/utils/host_utils.h

```diff
@@ -260,7 +260,22 @@ inline void safe_strcpy(char *cstr, const std::string &str, size_t limit, const
 }
 
 // MG param types
+/**
+  @brief Set multigrid parameters that are universal across Wilson-type
+         and staggered-type
+*/
 void setMultigridParam(QudaMultigridParam &mg_param);
```

Comment:
Does this declaration still need to be there.  E.g., should the Wilson and Staggered functions be the only functions exposed outside of set_params.cpp?

Reply:
Not sure at this point, I'll look back through it.